app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import (
+    NotWearingMaskError,
+    NotVaccinatedError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        datetime_today = datetime.date.today()
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        if visitor["vaccine"]["expiration_date"] < datetime_today:
+            raise OutdatedVaccineError
+        if not visitor["wearing_a_mask"]:
```

Comment:
Use .get because wearing_a_mask may not exist inside the visitor:
<img width="855" alt="image" src="https://github.com/mate-academy/py-lockdown-cafe/assets/65166203/d203f956-d275-4fbb-a5e4-e23b2ef3beee">